### PR TITLE
Generate cmd

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/NotificationCenter/metadata.js [hunk 1]
```diff
@@ -105,35 +94,37 @@ const ComponentWithIcon = ({ component }) => {
...
+  const defaultPath = 'ui/public/static/img/meshmodels/meshery-core/color/meshery-core-color.svg';
+
+  const [finalPath, setFinalPath] = useState(defaultPath);
 
   useEffect(() => {
     const loadImages = async () => {
-      if (imageCache[modelname]) {
-        setFinalPath(imageCache[modelname]);
-        return;
+      for (let i = 0; i < 2; i++) {
```
Comment (MUzairS15) [inline ui/components/NotificationCenter/metadata.js:109]: Does this loop through SVGs?

File: ui/components/NotificationCenter/metadata.js [hunk 2]
```diff
@@ -105,35 +94,37 @@ const ComponentWithIcon = ({ component }) => {
...
-  );
+  const defaultPath = 'ui/public/static/img/meshmodels/meshery-core/color/meshery-core-color.svg';
+
+  const [finalPath, setFinalPath] = useState(defaultPath);
 
   useEffect(() => {
     const loadImages = async () => {
-      if (imageCache[modelname]) {
-        setFinalPath(imageCache[modelname]);
-        return;
```
Comment (MUzairS15) [inline ui/components/NotificationCenter/metadata.js:122]: Why is this caching removed?

Comment (Jougan-0) [reply ui/components/NotificationCenter/metadata.js:122]: It was causing the last cached image for the entire model

File: ui/components/NotificationCenter/metadata.js (hunk 1 above)
Comment (Jougan-0) [reply ui/components/NotificationCenter/metadata.js:109]: Yes. You can upload the svg normally and it converts to path so you can directly upload the image.

**Comment by l5io** [comment]
🚀 Preview for commit 9fbeaaf6e980f0cfad83416ad63dee30a0601592 at: https://66fbd62d1ba99004d348c321--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/generating-models.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+
+**Example :**
+
+
+<pre class="codeblock-pre">
+<div class="codeblock"><div class="clipboardjs">mesheryctl model generate -f "git:://github.com/cert-manager/cert-manager/master/deploy/crds" -t template.json -r</div></div>
+</pre>
+
+
```
Comment (Raghav1783) [inline docs/pages/guides/configuration-management/generating-models.md:59]: we can also add an example for generating through the csv file.

<pre class="codeblock-pre">
<div class="codeblock"><div class="clipboardjs">mesehryctl model generate --f /path/to/csv-drectory</div></div>
</pre>

File: mesheryctl/internal/cli/root/registry/generate.go [hunk 4]
```diff
@@ -77,30 +60,13 @@ mesheryctl registry generate --registrant-def [path to connection definition] --
 // Generate a specific Model from a Google Spreadsheet (i.e. "Meshery Integrations" spreadsheet).
 mesheryctl registry generate --spreadsheet-id "1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw" --spreadsheet-cred --model "[model-name]"
 // Generate Meshery Models and Component from csv files in a local directory.
+mesheryctl registry generate --directory <DIRECTORY_PATH>
```
Comment (Jougan-0) [inline mesheryctl/internal/cli/root/registry/generate.go:63]:
```suggestion
```

**Comment by l5io** [comment]
🚀 Preview for commit 7b9da7dbd1a16f19bc2151c063c3da939dc54c3f at: https://67363c473dfa0c281f5bc03c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fcea395b9b5644e4538bd05ec03c64e12486dfcf at: https://67365759595a2e50f43ac981--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 571d6a91c2349a8d6a1828c76b9380ce446fc0cd at: https://673665fe2072ae8498838c2c--meshery-docs-preview.netlify.app

**Pull request merged**